multihist_plotter.C

```diff
@@ -0,0 +1,180 @@
+
+#include "tdrstyle.C"
+#include "TH1.h"
+#include "TH1F.h"
+
+int dolog=1;
+void multihist_plotter() 
```

Comment:
This could be made even easier to use:
```
void multihist_plotter(char* atitle, char* sigfile, std::vector<char*> sighnames, char* bkgfile, std::vector<char*> bkghnames)
```

Then on the command line:
```
root -l 'multihist_plotter.C("jet_alphamax","results_signal.root",{"bjet_alphamax","darkjet_alphamax"},"results_ttbar.root",{"jet_alphamax"}")'
```

This way, no need to edit the code to make a different histogram. This example could be added to the README.

Reply:
Done! I added some instructions on how to use multihist_plotter.C to the readme as well.

Reply:
Great! Can you check the plot margins (see below comment)? Then this will be ready to go in